.rhdh/docker/Dockerfile

```
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 #@follow_tag(registry.redhat.io/rhel9/go-toolset:latest)
-FROM registry.access.redhat.com/ubi9/go-toolset:1.20.12-3.1712567214 AS builder
+FROM registry.access.redhat.com/ubi9/go-toolset:1.21.10-1.1719562237 AS builder
```

```suggestion
FROM registry.access.redhat.com/ubi9/go-toolset:1.21.11-2 AS builder
```

There seems to be a new tag available. Can we use the most recent one instead?